doc/ug/getting_started_fpga.md

```diff
@@ -47,56 +48,85 @@ The `--build` argument is optional and ensures that the boot ROM is rebuilt for
+
+You can now use `dmesg` to determine which serial ports have been assigned.
+They should be named `'/dev/ttyACM*'`, e.g. `/dev/ttyACM1`.
+To ensure that you have sufficient access permissions to the devices, check `ls -l /dev/ttyACM*`.
+The udev rules given in the [Vivado installation instructions]({{< relref "install_instructions#xilinx-vivado" >}}) ensure this.
+
+## Connecting the Nexys Video board
 
 * Use a Micro USB cable to connect the PC with the *PROG*-labeled connector on the board.
 * Use a second Micro USB cable to connect the PC with the *UART*-labled connector on the board.
-* After connecting the UART, use `dmesg` to determine which serial port was assigned. It should be named `/dev/ttyUSB*`, e.g. `/dev/ttyUSB0`.
-* Ensure that you have sufficient access permissions to the device, check `ls -l /dev/ttyUSB*`. The udev rules given in the Vivado installation instructions ensure this.
+* After connecting the UART, use `dmesg` to determine which serial port was assigned.
+  It should be named `/dev/ttyUSB*`, e.g. `/dev/ttyUSB0`.
+* Ensure that you have sufficient access permissions to the device, check `ls -l /dev/ttyUSB*`.
+  The udev rules given in the [Vivado installation instructions]({{< relref "install_instructions#xilinx-vivado" >}}) ensure this.
 
 ## Flash the bitstream onto the FPGA
 
-To flash the bitstream onto the FPGA you need to use either the Vivado GUI or the command line.
+To flash the bitstream onto the FPGA you need to use either the command line or the Vivado GUI (Nexys Video board only).
+Depending on the FPGA device, the flashing itself may take several seconds.
+After completion, a message like this should be visible from the UART:
+
+```
+Version:    opentitan-snapshot-20191101-1-366-gca61d28
+Build Date: 2019-12-13, 13:15:48
+Bootstrap requested, initialising HW...
+HW initialisation completed, waiting for SPI input...
+```
 
-### Using the command line
+### Using the command line for the ChipWhisperer CW310 board
 
-Use the following command to program the FPGA with fusesoc.
+Use the following command to program the FPGA with the `cw310_loader` tool.
 
 ```console
-$ . /tools/xilinx/Vivado/2020.1/settings64.sh
 $ cd $REPO_TOP
-$ fusesoc --cores-root . pgm lowrisc:systems:chip_earlgrey_nexysvideo:0.1
+$ ./util/fpga/cw310_loader.py --bitstream build/lowrisc_systems_chip_earlgrey_cw310_0.1/synth-vivado/lowrisc_systems_chip_earlgrey_cw310_0.1.bit
 ```
 
-This should produce a message like this from the UART:
+### Using the command line for the Nexys Video board
 
-```
-Version:    opentitan-snapshot-20191101-1-366-gca61d28
-Build Date: 2019-12-13, 13:15:48
-Bootstrap requested, initialising HW...
-HW initialisation completed, waiting for SPI input...
+Use the following command to program the FPGA with fusesoc.
+
+```console
+$ . /tools/xilinx/Vivado/2020.1/settings64.sh
+$ cd $REPO_TOP
+$ fusesoc --cores-root . pgm lowrisc:systems:chip_earlgrey_nexysvideo:0.1
```

Comment:
Not that it matters any more, but the recommended way of flashing for Nexys Video is the ot-pgm-fpga tool.

Reply:
I think it does matter. I've changed the instructions to refer to the `opentitan-pgm-fpga` tool. Thanks for pointing this out.